### PR TITLE
[PMK-2565 + PMK-2601] Tenants & Users changes and fixes

### PR DESCRIPTION
**Pull request by xantorres**
Also done some major homogeneization and simplification of the form stylings thorough the app.

### CODE REVIEW

File: src/app/core/components/validatedForm/FormFieldCard.tsx [hunk 1]
```diff
@@ -4,18 +4,36 @@ import { makeStyles } from '@material-ui/styles'
...
+}
+const defaultMaxWidth = 715
+
+export const useStyles = makeStyles<Theme, ContainerProps>((theme) => ({
   root: {
+    maxWidth: ({ maxWidth = defaultMaxWidth }) => maxWidth,
     borderRadius: 4,
     boxShadow:
-      '0 2.5px 2.5px -1.5px rgba(0, 0, 0, 0.2), 0 1.5px 7px 1px rgba(0, 0, 0, 0.12), 0 4px 5px 0.5px rgba(0, 0, 0, 0.14)',
+      '0 2.5px 2.5px -1.5px rgba(0, 0, 0, 0.2), ' +
```
Comment (jruddell) [inline src/app/core/components/validatedForm/FormFieldCard.tsx]: would a string template be more appropriate here?

File: src/app/core/components/validatedForm/ValidatedForm.js [hunk 2]
```diff
@@ -202,13 +207,43 @@ class ValidatedForm extends PureComponent {
...
+          {nowrap ? (
+            inputs
+          ) : (
+            <FormFieldCard
+              title={title}
+              link={link}
+              topContent={topContent}
+              className={className}
+              maxWidth={maxWidth}
+              inputsWidth={inputsWidth}
```
Comment (jruddell) [inline src/app/core/components/validatedForm/ValidatedForm.js]: how do we handle content of varying size? for instance in this screenshot, the checkbox below the table is half width, the table though is full width. both in the same card
image

File: src/app/core/components/validatedForm/FormFieldCard.tsx (hunk 1 above)
Comment (xantorres) [reply src/app/core/components/validatedForm/FormFieldCard.tsx]: I don't want to have a multilined attribute value here, I just want to be able to see what its meant to be in one line at a glance (as it originally was), so no multilined template is needed here.

File: src/app/core/components/validatedForm/ValidatedForm.js (hunk 2 above)
Comment (xantorres) [reply src/app/core/components/validatedForm/ValidatedForm.js]: Thanks for noticing that, I fixed it by changing the input style itself, probably should check other types of inputs too though.

image

File: src/app/core/components/validatedForm/CheckboxField.js [hunk 3]
```diff
@@ -5,11 +5,13 @@ import { FormControl, FormControlLabel, FormHelperText } from '@material-ui/core
...
 import { withInfoTooltip } from 'app/core/components/InfoTooltip'
 import { compose } from 'app/utils/fp'
-import withFormContext, {
-  ValidatedFormInputPropTypes,
-} from 'core/components/validatedForm/withFormContext'
+import withFormContext, { ValidatedFormInputPropTypes } from 'core/components/validatedForm/withFormContext'
 
 const styles = (theme) => ({
+  root: {
+    minWidth: '50%',
```
Comment (jruddell) [inline src/app/core/components/validatedForm/CheckboxField.js]: Not a fan of percentage widths. What are we trying to accomplish here? Why not just use a class and pass to the form elements that we want to style differently?

File: src/app/core/components/validatedForm/ValidatedForm.js [hunk 4]
```diff
@@ -202,13 +208,41 @@ class ValidatedForm extends PureComponent {
...
+      debug,
+      id,
+      title,
+      link,
+      className,
+      topContent,
+      formActions,
+      maxWidth,
+      inputsWidth,
+      nowrap,
```
Comment (jruddell) [inline src/app/core/components/validatedForm/ValidatedForm.js]: We should call this something more specific like `elevated` or `asCard`. nowrap sounds more like wrapping content to a new line `flex-direction: nowrap`

File: src/app/core/components/validatedForm/CheckboxField.js (hunk 3 above)
Comment (xantorres) [reply src/app/core/components/validatedForm/CheckboxField.js]: The 50% width is a way to give enough space to the information tooltip that has to be shown on the right.

File: src/app/core/components/validatedForm/ValidatedForm.js (hunk 4 above)
Comment (xantorres) [reply src/app/core/components/validatedForm/ValidatedForm.js]: `elevated` sounds good

**Pull request merged**